src/components/PupilComponents/QuizResultMCQ/QuizResultMCQ.stories.tsx

```diff
@@ -42,6 +42,7 @@ export const CorrectSingle: Story = {
   args: {
     answers: mcqTextAnswers,
     feedback: [null, null, "correct", null],
```

Comment:
I think we established that this state is impossible because the feedback for mcqAnswers is always marked as correct or incorrect irrespective of whether the pupil clicked on it. So we should implement all of these stories and tests to that effect.